### ISSUE TITLE
 [Accessibility : Alternative content] Filter field in network page does not have alternative content 

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/kimchi#890]
Filter field in Network page does not have alternative content
